pandas/io/formats/format.py

```diff
@@ -252,8 +252,15 @@ def _get_formatted_index(self):
 
     def _get_formatted_values(self):
         values_to_format = self.tr_series._formatting_values()
-        return format_array(values_to_format, None,
-                            float_format=self.float_format, na_rep=self.na_rep)
+        if self.index:
+            return format_array(values_to_format, None,
+                                float_format=self.float_format,
+                                na_rep=self.na_rep)
+        else:
+            return format_array(values_to_format, None,
+                                float_format=self.float_format,
+                                na_rep=self.na_rep,
+                                leading_space=False)
```

Comment:
Just `leading_space=self.index` then you don't need to duplicate the call?

Reply:
hmm... not sure about this... `self.index` by default is `True`, not `None`, right?

Reply:
You can do `leading_space=bool(self.index)` if you want.